docs/providers/aws/events/apigateway.md

```diff
@@ -1476,3 +1476,20 @@ provider:
 ```
 
 The log streams will be generated in a dedicated log group which follows the naming schema `/aws/api-gateway/{service}-{stage}`.
+
+By default, API Gateway access logs will use the following format:
+
+```
+'requestId: $context.requestId, ip: $context.identity.sourceIp, caller: $context.identity.caller, user: $context.identity.user, requestTime: $context.requestTime, httpMethod: $context.httpMethod, resourcePath: $context.resourcePath, status: $context.status, protocol: $context.protocol, responseLength: $context.responseLength'
+```
+
+You can specify your own [format for API Gateway Access Logs](https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-logging.html#apigateway-cloudwatch-log-formats) by including your preferred string in the `logFormat` property:
+
+```yml
+# serverless.yml
+provider:
+  name: aws
+  logs:
+    restApi:
+      logFormat: '{ "requestId":"$context.requestId",   "ip": "$context.identity.sourceIp" }'
```

Comment:
I wonder whether `format` would be better. as we're already in `logs` context.

Currently it's `logs.restApi.logFormat`, wouldn' it be cleaner to have `logs.restApi.format` ?

Reply:
Yep, I'm good with that. Seems repetitive. Will update.